synapse/api/ratelimiting.py

```diff
@@ -23,10 +23,11 @@ class Ratelimiter(object):
     def __init__(self):
         self.message_counts = collections.OrderedDict()
 
-    def send_message(self, user_id, time_now_s, msg_rate_hz, burst_count, update=True):
+    def send_message(self, key, time_now_s, msg_rate_hz, burst_count, update=True):
```

Comment:
Can we rename the function too? Like maybe to `can_do_action` please? And `msg_rate_hz` to just `rate_hz`.

Reply:
Sure, makes sense.